pkg/backend/benchmark_test.go

```diff
@@ -119,22 +127,28 @@ func BenchmarkRendersStorm(b *testing.B) {
 	n := 50
 	errs := make(chan []error, n)
 
-	for i := 0; i < b.N; i++ {
-		for j := 0; j < 50; j++ {
-			wg.Add(1)
-			go func() {
-				defer wg.Done()
-				_, err := Renders(ctx, backends, types.NewRenderRequest(nil, 0, 0))
-				errs <- err
-			}()
-		}
-
-		wg.Wait()
-		for j := 0; j < n; j++ {
-			err := <-errs
-			if len(err) != 0 {
-				b.Fatal(err)
+	consistencyChecks := []bool{false, true}
```

Comment:
Yes and there is a significant difference (which is about 3 to 4 times slower on storm bench) but it does not necessarily hurt the real result because it is not quite considerable compared to latency of fetching metrics from backends.